src/csharp/Microsoft.Spark/Utils/AssemblyLoader.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.Spark.Utils
+{
+    internal class AssemblyLoader
+    {
+        internal static Func<string, Assembly> LoadFromFile { get; set; } = Assembly.LoadFrom;
+
+        internal static Func<string, Assembly> LoadFromName { get; set; } = Assembly.Load;
+
+        internal static readonly ConcurrentDictionary<string, string> s_assemblyNameToFileName =
```

Comment:
(nit) use a property and don't use `s_` prefix.

Reply:
removed field.